.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,181 @@
+name: Benchmark Xdebug performance
+
+on:
+    schedule:
+        # Every Sunday at 03:00 UTC
+        - cron: '0 3 * * 0'
+    workflow_dispatch: 
+    pull_request:
+        types: [opened, edited, reopened, synchronize]
+
+jobs:
+    run-benchmark:
+        if: |
+            github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' ||
+            (
+                github.event_name == 'pull_request' &&
```

Comment:
For pull requests, we check that the pull request title starts with a specific prefix so that we don't trigger it for every single pull request